server/stt_metrics_test.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func TestSTTMetrics(t *testing.T) {
 	t.Run("StartSTTChecksAndRecordMetrics", func(t *testing.T) {
-		// TODO: Remove this once the test environment has been setup properly
+		// Remove this once the test environment has been setup properly
```

Comment:
No, please keep TODO (without typo) – it is real, and we use them for tracking

Reply:
done